.travis.yml

```diff
@@ -44,3 +44,5 @@ matrix:
 
 addons:
   postgresql: '9.4'
+
+sudo: false
```

Comment:
hmm, actually, it never uses it because the code switching the Postgres version based on the ENV variable is broken and never switches it (meaning we don't actually test against multiple versions)